persistent/Database/Persist.hs

```diff
@@ -40,6 +62,26 @@ infixr 3 =., +=., -=., *=., /=.
   forall v typ.  PersistField typ => EntityField v typ -> typ -> Update v
 
 -- | Assign a field a value.
+--
+-- === __Example usage__
+--
+-- @
+-- updateAge :: MonadIO m => ReaderT SqlBackend m ()
+-- updateAge = updateWhere [UserName ==. \"SPJ\" ] [UserAge =. 45]
+-- @
+--
+-- Similar to `updateWhere` you can use other functions present in the module "Database.Persist.Class". Note that the first attribute of `updateWhere` is [`Filter` val] and second attribute is [`Update` val]. By comparing this with the type of `==.` and `=.`, you can see that they match up in the above usage.
```

Comment:
> Similar to `updateWhere` you can use other functions present in the module "Database.Persist.Class".

I get what it means now, but I initially was confused by this sentence and didn't understood how it related to `=.`.

> Note that the first attribute of `updateWhere` is [`Filter` val] and second attribute is [`Update` val].

"attribute" should probably be "parameter" or "argument" here